packages/react/src/components/Table/TableHead/TableHead.test.jsx

```diff
@@ -43,23 +43,47 @@ jest.mock('lodash/debounce', () => (fn) => fn);
 
 describe('TableHead', () => {
   it('be selectable by testID or testId', () => {
-    const { rerender } = render(<TableHead {...commonTableHeadProps} testID="TABLE_HEAD" />, {
-      container: document.body.appendChild(document.createElement('table')),
-    });
+    const columns = commonTableHeadProps.columns.map((c) => ({
+      ...c,
+      overflowMenuItems: [
+        {
+          id: '1',
+          text: 'one',
+        },
+        {
+          id: '2',
+          text: 'two',
+        },
+      ],
+    }));
+    const { rerender } = render(
+      <TableHead {...commonTableHeadProps} columns={columns} testID="TABLE_HEAD" />,
+      {
+        container: document.body.appendChild(document.createElement('table')),
+      }
+    );
 
     expect(screen.getByTestId('TABLE_HEAD')).toBeDefined();
     expect(screen.getByTestId('TABLE_HEAD-column-col1')).toBeDefined();
     expect(screen.getByTestId('TABLE_HEAD-column-col2')).toBeDefined();
     expect(screen.getByTestId('TABLE_HEAD-column-col3')).toBeDefined();
+    expect(screen.getAllByTestId('table-head--overflow')).toHaveLength(3);
```

Comment:
why is this not `TABLE_HEAD--overflow`?

Reply:
b/c it's a hard-coded testId that's existed for a long time. It'll need to be replaced/changed in v3.